src/_playground/documentation/utils/SetMeta.js

```diff
@@ -0,0 +1,21 @@
+import Helmet from 'react-helmet-async';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const SetMeta = ({ title, description }) => {
+    return (
+        <Helmet
+            defaultTitle='Fundamental React'
+            titleTemplate='Fundamental React - %s'>
+            <title>{title}</title>
+            <meta content={description} name='description' />
```

Comment:
For pages without a description (_e.g._ Home), it is emitting:
```
<meta content="" name="description" data-rh="true">
```
Should we only output this if a description exists?

Reply:
Also, this is outputting markdown source (if it was supplied), which might be okay and might be problematic.  See:
```
<meta content="The **Calendar** is commonly used as the contents of a **Popover** when composing a **Date Picker**.
                It is rarely used on its own as a standalone component." name="description" data-rh="true">
```
Is there a way we can quickly cleanse this output?

Reply:
For documentation, @greg-a-smith and I talked offline and decided to have a followup story to add a description meta tag to the head as it's not strictly necessary at the moment.